src/watson/registration.py

```diff
@@ -52,6 +52,12 @@ def _resolve_field(self, obj, name):
         """Resolves the content of the given model field."""
         name_parts = name.split("__", 1)
         prefix = name_parts[0]
+        # Check if the attribute exists in the database
+        try:
+            if not getattr(obj, prefix):
```

Comment:
The problem is that it doesn't throw the AttributeError what hasattr() should do. It throws the ObjectDoesNotExist error. I looked at the object where the error were thrown and the object definitely had the chosen attribute. After django-watson wanted to access the attribute django has thrown the ObjectDoesNotExist Exception because the attribute hadn't the entry in the database.